Documentation/CRDs/Cluster/external-cluster.md

```diff
@@ -140,35 +159,38 @@ export RGW_POOL_PREFIX=default
     rook-ceph-external   /var/lib/rook                162m   Connected   HEALTH_OK
     ```
 
-6. StorageClass will also be created, verify its creation. `ceph-rbd` and `cephfs` StorageClass would be respective name for RBD and CephFS StorageClass.
+6.  Verify the creation of the storage class depending on the rbd pools and filesystem provided.
+    `ceph-rbd` and `cephfs` would be the respective names for the RBD and CephFS storage classes.
     ```console
     kubectl -n rook-ceph-external get sc
     ```
 
 7. Then you can now create a [persistent volume](https://github.com/rook/rook/tree/master/deploy/examples/csi) based on these StorageClass.
 
-### CephCluster example (management)
-
-The following CephCluster CR represents a cluster that will perform management tasks on the external cluster.
-It will not only act as a consumer but will also allow the deployment of other CRDs such as CephFilesystem or CephObjectStore.
-You would need to inject the admin keyring for that.
-
-The corresponding YAML example:
-
-```yaml
-apiVersion: ceph.rook.io/v1
-kind: CephCluster
-metadata:
-  name: rook-ceph-external
-  namespace: rook-ceph-external
-spec:
-  external:
-    enable: true
-  dataDirHostPath: /var/lib/rook
-  cephVersion:
-    image: quay.io/ceph/ceph:v17.2.6 # Should match external cluster version
+### Connect to an External Object Store
```

Comment:
This section is needed if installing with the helm chart, right? Up in the helm install section, how about a comment that says they need to continue with this section?

Reply:
No even this is needed with normal manifest installation

Reply:
Yes, it's needed for both manifest and helm installs. To clarify, I was suggesting that in the helm section we need to tell them to continue with this section, otherwise they might miss it.